### ISSUE TITLE
Adding Some Helpful Resources Links below the Syntax Pad

### ISSUE THREAD

**Issue by vrii14** [kothariji/SyntaxMeets#37]
The area under syntax pad and besides input output block looks incomplete and empty.

**To modify the same I would like to create this issue**
I can add some helpful resource links like GFG, Hackkerrank, LeetCode since programmers who are coding might need these resources frequently and it will also fill up the place nicely.

If you like the idea, please assign me this issue for GSSOC'21
![Screenshot from 2021-03-10 10-05-56](https://user-images.githubusercontent.com/56122560/110577313-67169280-8188-11eb-8160-fce9b1a3c762.png)
The bottom right area is where I'm referring to

**Comment by vrii14** [comment]
Thanks @kothariji!  So, basically I meant something in a block like i/o with a heading "Some Useful Resources" and in the block icons with their website links like GFG tutorial section link, CodeChef and Hackerrank learning pages links with their icons and we can add some more to it later!

**Comment by kothariji** [comment]
Yes @vrii14 , but do you think it's a right call to advertise websites of other coding platforms, and that to static, means it will be there always.
Instead, here are some cool stuff which I think we can add
1. ranodm motivational/coding quotes or facts
2. chrome dino game, so that if a user is bored he can play that 😆 
3. Random algorithm with there Wikipedia link, so a user can click on it and explore more.

Let me know what you think about this.

**Comment by Vishv07** [comment]
@kothariji  Agree with you! Bcz this isn't a CP platform.

**Comment by Priyankcoder** [comment]
@kothariji I think adding a random motivational quote would be a great idea. As sometimes coding is tiresome and a glance at the quote might help stick to coding. Chrome Dino game is cool too but I think it will distract users from coding. 
I can implement a random motivation quote feature if you assign this task to me.

**Comment by vrii14** [comment]
Yess @kothariji 
I think random motivational / coding quotes would be a great idea and we could also add some cool transitions for changing quotes too!

**Comment by kothariji** [comment]
Yes, but actually we are working on a video call feature in issue #21  so because of that there might be some UI changes in the Room. So I think we should wait for that issue to be resolved, and we can discuss this issue if there is any vacant area left.

**Comment by vrii14** [comment]
Okay sure!

**Comment by kothariji** [comment]
hi @vrii14, you can start working on this issue, Currently, we decided to keep issue #21 on hold, so yes we can work on this part.
Also before going to code part, please send us a prototype design of how are you thinking to implement the component.

**Comment by vrii14** [comment]
Okay Sure @kothariji. I will send a prototype as soon as possible

**Comment by kothariji** [comment]
Another cool thing triggered in my mind.
We can add a small music player below SyntaxPad. Share your views on this 🙌

**Comment by subhangi2731** [comment]
please assign me this

**Comment by Furquan712** [comment]
I would like to work on this, will you assign me?

**Comment by vrii14** [comment]
@kothariji I apologize for not contributing or giving an update. Had some personal issues. I can start working on this in a day or two or if you want to assign it to someone else that's okay too.

**Comment by vrii14** [comment]
@kothariji 
Hi, so I haven't used canva earlier so I wasn't sure why was the golden border not appearing on all the sides. But basically this could be the dsign.  
The quote will be randomly fetched from [this](https://zenquotes.io/) API along with pre-formatted HTML and author, And the HAPPY CODING! will be static.The quote can be updated in a fixed amount of time with some CSS transition.  
I could also add something more but that would make it too much for the limited space. Let me know your thoughts on this!
![canva](https://user-images.githubusercontent.com/56122560/117686311-5dcf9380-b1d4-11eb-9d01-db1e88338797.png)

**Comment by vrii14** [comment]
If it's going to use Spotify API would something like this work?
![music](https://user-images.githubusercontent.com/56122560/117836948-a5b6ef00-b296-11eb-8d91-ab7a71d3a7ea.png)

**Comment by kothariji** [comment]
> Oh, that sounds cool! So these songs should be randomly fetched on refreshing via Spotify API or something? Is that what you have in mind or something else?
> I'll try to create a design for this and will let you know.

Yes, something like that, we can fetch songs by any OPEN SOURCE API.

**Comment by kothariji** [comment]
> If it's going to use Spotify API would something like this work?
> ![music](https://user-images.githubusercontent.com/56122560/117836948-a5b6ef00-b296-11eb-8d91-ab7a71d3a7ea.png)

@vrii14, awesome ✨  this is exactly, what we are trying to achieve. Though there would be some design changes. I'll send you a prototype design once I discuss it with the team. 🙌

**Comment by vrii14** [comment]
> > If it's going to use Spotify API would something like this work?
> > ![music](https://user-images.githubusercontent.com/56122560/117836948-a5b6ef00-b296-11eb-8d91-ab7a71d3a7ea.png)
> 
> @vrii14, awesome  this is exactly, what we are trying to achieve. Though there would be some design changes. I'll send you a prototype design once I discuss it with the team.

Okay sure!

**Comment by kothariji** [comment]
@vrii14 
![image](https://user-images.githubusercontent.com/60866749/118457837-cb6d4980-b717-11eb-9d23-724375ef9b5f.png)
or any other 3rd party npm library would work great.
please let me know about the same 🙌

**Comment by vrii14** [comment]
@kothariji Sure I'll look into some npm libraries and let you know.  
And actually, I am in the middle of end semester exams so there was a delay in seeing this.

**Comment by kothariji** [comment]
> @kothariji Sure I'll look into some npm libraries and let you know.
> And actually, I am in the middle of end semester exams so there was a delay in seeing this.

No hurries @vrii14, you can contribute after the exams. Also, all the best for the exams 🙌

**Comment by vrii14** [comment]
> > @kothariji Sure I'll look into some npm libraries and let you know.
> > And actually, I am in the middle of end semester exams so there was a delay in seeing this.
> 
> No hurries @vrii14, you can contribute after the exams. Also, all the best for the exams

Thanks, @kothariji.  
I found these two npm libraries useful and interesting. You can look through them
[react-jinke-music-player](https://www.npmjs.com/package/react-jinke-music-player)  
[react-audio-player](https://www.npmjs.com/package/react-audio-player)

**Comment by kothariji** [comment]
@vrii14 I think react-jinke-music-player will do the job. what say?

**Comment by vrii14** [comment]
@kothariji Cool, I'll give it a shot as soon as I am done with my exams

**Comment by kothariji** [comment]
sure @vrii14 👍✨
